packages/babel-plugin-transform-es2015-modules-commonjs/test/fixtures/strict/export-all/expected.js

```diff
@@ -3,15 +3,51 @@
 Object.defineProperty(exports, "__esModule", {
   value: true
 });
+var _exportNames = {
+  z: true,
+  a: true,
+  b: true,
+  d: true,
+  e: true,
+  f: true,
+  c: true
+};
+exports.b = b;
+exports.default = _default;
+Object.defineProperty(exports, "c", {
+  enumerable: true,
+  get: function () {
+    return _mod.c;
+  }
+});
+exports.f = exports.e = exports.d = exports.a = exports.z = void 0;
 
 var _mod = require("mod");
 
 Object.keys(_mod).forEach(function (key) {
   if (key === "default" || key === "__esModule") return;
+  if (Object.prototype.hasOwnProperty.call(_exportNames, key)) return;
```

Comment:
Shower thought: isn't `Object.prototype.hasOwnProperty` `===` `Object.hasOwnProperty` 🤔

Reply:
Yeah agreed, I just find this easier to follow.